service/device/registry/test-steps/src/main/java/org/eclipse/kapua/service/device/registry/steps/DeviceRegistrySteps.java

```diff
@@ -2559,4 +2560,65 @@ public void iFindDeviceWithClientId(String deviceName) {
 
         assertEquals(device.getClientId(), deviceName);
     }
+
+    @And("^I add device to group$")
+    public void iAddDeviceToGroup() throws Exception{
```

Comment:
Change method name to: "public void iAddDeviceToGroup(String deviceName, String groupName) throws Exception{